appcelerator/titanium_mobile
# feat(android): use example platform folder for module builds

When you build a module with a custom build.gradle you couldn't use the `/example` folder as it didn't copy the build.gradle to the correct place.

This PR will copy a `/platform` folder into the tmp build folder at the correct place so you can use a custom `build.gradle` for your module. Like https://github.com/m1ga/ti.exoplayer/tree/main/example

## Test:

* clone https://github.com/m1ga/ti.exoplayer/
* go to `/android`
* run `ti build -p android` to build the module + the example app
* it should start the example app (before it complaint because it didn't use the build.gradle)

## Test 2:
* create a new module and build it (default app.js)
